test/src/test/java/hudson/model/FingerprintTest.java

```diff
@@ -23,6 +23,7 @@
  */
 package hudson.model;
```

Comment:
Please also move the test to the jenkins package.

Reply:
Hi @StefanSpieker thanks for reviewing :)
FingerprintTest is for Fingerprint class, which resides in hudson.model . Should it also be moved to Jenkins package?

Reply:
Sorry, my bad, then please keep it there.

Reply:
No worries, thanks for reviewing :)